### ISSUE TITLE
Ads: Accessibility Swarm

### ISSUE THREAD

**Issue by tochwill** [bbc/simorgh#5956]
**Is your feature request related to a problem? Please describe.**
Carry out an accessibility swarm on the ads component, for both AMP and canonical. Go through this [checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face. 

This component should NOT be visible to screen readers.

**Describe the solution you'd like**
Given I use a screen reader
When there is advertising on the page
Then I cannot navigate to any advertising
And none of the advertising content is read out (including the ‘ADVERTISEMENT’ label)

Given I use a keyboard
When there is advertising on the page
Then all links and buttons within any advertising must not be focusable
And I cannot reach any advertising content by pressing the TAB key

Given there is advertising on the page
When the ‘ADVERTISEMENT’ label is displayed
Then this must meet the colour contrast ratio of 4.1:5 (as this is key to understanding)

Given I am on a page with advertising
When a leaderboard advert is displayed
Then it is displayed above the main content of the page, and bypassed when using the skip to content link

Given I am on a page with advertising
When a leaderboard advert is displayed
Then it is within its own section, outside of the main element of the page

Given I am on a page with advertising
When a multiple adverts are displayed
Then they are within their own section, identified by distinct labels (this could be an aria-label)

**Comment by tochwill** [comment]
I'm just taking a closer look at the top two points:

**Screenreader navigation**
Is it enough to set the whole ad container as `aria-hidden` to hide this from screenreaders?

**Tab Order**
Is it possible to remove the advertisement from the tab order? We do not control content within the iframe that is loaded in, and the parent container's tabindex won't be inherited by its children, if I understand correctly? There is going to be an `a` tag with a link to the ad's destination. https://www.theguardian.com/uk uses AMP ads for the leaderboard at the top of the page, the behaviour is the same here. Would welcome any recommendations for how we could remove the content of the ad from the tab order!

For canonical ads, we need check the accessibility requirements through with GNL and their provided solution.

**Comment by greenc05** [comment]
Setting aria-hidden isn't enough to fully hide content from screen readers. Focusable elements will still be reachable. 

GNL add the tabindex attributes at their end on responsive news. I assume they can do the same this time? With regards to AMP, if we/someone can't add tabindex -1 to all focusable elements such as iframes, links, buttons then we'll have to settle for just aria-hidden on amp.

**Comment by tochwill** [comment]
@tochwill update this issue with a link to accessibility criteria and additions from example mapping.

**Comment by DenisHdz** [comment]
We've found the following issues during the a11y swarm:

- **P1**: The font size of “PUBLICIDAD” is not increased even when font resized to largest. 
    This is because the link is using `px`s rather than `rem`s.

- **P2**: The link for the advert inside the iframe does not have `tabindex=-1` applied by GNL. 
   See bug in cell 38/J:
   ```
   When swiping through links, the ad unit is selected in the tab order for both Leaderboards and 
   MPUs. It selects just the ad unit itself and not the advertisement label.
  ```
  This is probably as a result of cell 24/D.

This will be brought up with GNL, since we can't fix it ourselves.
